lib/player.js

```diff
@@ -3646,6 +3648,39 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
           'streaming.preferNativeHls instead.');
     }
 
+    // map liveSyncMinLatency and liveSyncMaxLatency to liveSyncTargetLatency
+    // if liveSyncTargetLatency isn't set.
```

Comment:
This works for mapping the values to target latency but won't work well if each option is set individually.
If I don't delete the config option as part of deprecation, I could probably default to `this.config_.streaming.liveSyncMinLatency` etc, thoughts?

Reply:
It's a risk we can take. I would leave it as is